<launch>
  <node name="cugo_controller" pkg="CuGoPy_Controller" type="cugo_controller" output="screen" >
    <param name="timeout" value="0.05" />
    <param name="wheel_radius_l" value="0.03858" />
    <param name="wheel_radius_r" value="0.03858" />
    <param name="tread" value="0.380" />
    <param name="reduction_ratio" value="1.0" />
    <param name="encoder_resolution" value="2048" />
    <param name="encoder_max" value="2147483647" />
    <param name="arduino_port" value="8888" />
    <!--<param name="arduino_addr" value="192.168.8.216" />-->
    <param name="arduino_addr" value="127.0.0.1" />
    <param name="odom_frame_id" value="odom" />
    <param name="odom_child_frame_id" value="base_link" />
  </node>
</launch>
